pkg/console/install_panels.go

```diff
@@ -1263,6 +1265,17 @@ func addInstallPanel(c *Console) error {
 				}
 				logrus.Info("Local config (merged): ", c.config)
 			}
+
+			// Adding default NIC bonding options for every network config if missing
+			for netName, network := range c.config.Install.Networks {
+				if network.BondOptions == nil {
+					msg := fmt.Sprintf("Adding default NIC bonding options for \"%s\"", netName)
+					logrus.Info(msg)
+					printToPanel(c.Gui, msg, installPanel)
+					c.config.Install.Networks[netName] = network.AddDefaultBondOptions()
+				}
```

Comment:
There is a subtle that not every NIC config is a bond. We may not need this *for every network* but management NIC only, though it wouldn't affect anything if a `Network` doesn't contain any interfaces with bond options on it.

Reply:
You may want to check the length of `network.interfaces` to determine this behavior

Reply:
@weihanglo What do you think about moving default options to here? https://github.com/harvester/harvester-installer/blob/ff0bd2f4a11e24f5d762546399c93c0c7a2f7337/pkg/config/cos.go#L368